api/controllers/SearchController.js

```diff
@@ -10,6 +10,14 @@ var striptags = require('striptags');
 var querystring = require('querystring');
 var numeral = require('numeral');
 
+isInPackageSearch = function(query){
```

Comment:
be careful to declare them with `var`, otherwise this will live in the global scope